CONTRIBUTING.md

```diff
@@ -51,8 +51,13 @@ following pattern:
 Use these:
 - [core] Feature for Kibicara core
 - [frontend] Feature for Kibicara frontend
-- [platform] Feature for platforms like Twitter, Telegram, ...
+- [<platform>] Feature for platforms, e.g.
+    - [twitter]
+    - [telegram]
+    - [email]
+    - ...
 - [tests] Tests
+- [doc] Documentation
```

Comment:
Please remove this line since we already have misc for doc

Reply:
I think misc and doc should be two separate things. Documentation isn't really miscellaneous.